Example/ReusableDemo iOS/CollectionViewCells/CollectionHeaderView.swift

```diff
@@ -20,7 +20,7 @@ import Reusable
  * Which in fact is just a convenience typealias that combines
  * `NibLoadable` & `Reusable` protocols.
  */
-final class CollectionHeaderView: UICollectionReusableView, NibReusable {
+final class CollectionHeaderView: UICollectionReusableView, BundledSelf, NibReusable {
```

Comment:
nitpick: for consistency with other examples, what do you think about putting `BundledSelf` after `NibReusable` instead of before? This is not a big deal and doesn't change the behaviour, but I just thought it would be nice to align with the rest 🙂